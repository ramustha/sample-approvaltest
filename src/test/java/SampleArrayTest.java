import org.approvaltests.Approvals;
import org.junit.Test;

import java.util.Arrays;

public class SampleArrayTest {

  @Test
  public void testList() {
    String[] names = {"Llewellyn", "James", "Dan", "Jason", "Katrina"};
    Arrays.sort(names);

    Approvals.verifyAll("Test List of Name", names);
  }

}
