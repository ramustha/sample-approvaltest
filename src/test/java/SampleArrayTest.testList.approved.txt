Test List of Name[0] = Dan
Test List of Name[1] = James
Test List of Name[2] = Jason
Test List of Name[3] = Katrina
Test List of Name[4] = Llewellyn
